libp2p/protocols/pubsub/pubsubpeer.nim

```diff
@@ -53,10 +56,16 @@ type
   DropConn* = proc(peer: PubSubPeer) {.gcsafe, raises: [].} # have to pass peer as it's unknown during init
   OnEvent* = proc(peer: PubSubPeer, event: PubSubPeerEvent) {.gcsafe, raises: [].}
 
+  Ttlmessage* = object
+    msg*: seq[byte]
+    ttl*: Moment
```

Comment:
I think `ttl` doesn't fit the usage we're making of it. To me, `ttl` suggests either the amount of time the message is allowed to live (time increment, e.g.: `5s`) or the timestamp when the message should expire.

Given you are saving the message generation timestamp (which I fully agree with), I'd use a different name as that might lead to confusion.

Reply:
Right, I think I'll change it to `timestamp`.

Reply:
I would argue, though, that "timestamp" can be inferred from the type (`Moment`). I would use something like `created` or `createdAt`, because `timestamp` alone could lead to a missunderstanding between "creation timestamp" and "deletion date".

Reply:
I thought about `created`, but it'd also be misleading, that's not the time when the message was created, but added to the queue. How about: 

```
  QueuedMessage* = object
    msg*: seq[byte]
    addedAt*: Moment
```

Reply:
Right, agree. Said that one just because I'm used to the SQL standard. The one you said works perfectly, very explicit :)